src/main/java/com/arpnetworking/commons/math/KahanAccumulator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Dropbox
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.arpnetworking.commons.math;
+
+/**
+ * Accumulator implementation using Kahan Sum algorithm. This has O(1) error.
```

Comment:
super nit. "This has O(1) error" => "This has __effectively__ O(1) error"

This is because for double precision floating point numbers, n would have to be larger than ~10^16 for the nε^2 term in the error to start becoming significant, but it still isn't constant

Reply:
Done!